build/initialize-pipeline.ps1

```diff
@@ -8,21 +8,35 @@ $sourceBranch = $env:BUILD_SOURCEBRANCH
 Write-Host "BUILD_REASON: '$buildReason'"
 Write-Host "BUILD_SOURCEBRANCH: '$sourceBranch'"
 
+# See if artifacts should be built
 if ($buildReason -eq "PullRequest") {
+
+  # first we see if [pack] is in the PR title.
+  # This often gets rate limited, so it isn't reliable.
+
+  $pack = $false
   try {
-    # parse PR title to see if we should pack this
     $response = Invoke-RestMethod api.github.com/repos/$env:BUILD_REPOSITORY_ID/pulls/$env:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER
     $title = $response.title.ToLowerInvariant()
     Write-Host "Pull request '$title'"
-    if ($title.Contains("[pack]")) {
-      Write-Host "##vso[task.setvariable variable=BuildArtifacts;isOutput=true]true"
-      Write-Host "Setting 'BuildArtifacts' to true."
-    }
+    $pack = $title.Contains("[pack]")
   }
   catch {
-    Write-Warning "Failed to get pull request title. Artifacts will not be built or packed."
+    Write-Warning "Failed to get pull request title."
     Write-Warning $_
   }
+
+  # Next we check if the commit message contains '--pack'
+  $commitMessage = $env:BUILD_SOURCEVERSIONMESSAGE
```

Comment:
Can we track documenting this?

Reply:
Sure - but this will be made obsolete with 1ES work.

Reply:
The first thing that comes to mind is Engineering Hub, but if this is truly temporary, it does become less important.